dd-sdk-android/src/main/json/log/log-schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "log-schema.json",
+  "title": "LogEvent",
+  "type": "object",
+  "description": "Structure holding information about a Log",
+  "properties": {
+    "status": {
+      "type": "string",
+      "description": "The severity of this log",
+      "enum": [
+        "critical",
+        "error",
+        "warn",
+        "info",
+        "debug",
+        "trace",
+        "emergency"
+      ],
+      "readOnly": true
```

Comment:
Since this is an enum, I think we can let customer change the severity (especially for automatic logs)